Agrume/AgrumeCell.swift

```diff
@@ -60,12 +61,24 @@ final class AgrumeCell: UICollectionViewCell {
   // if set to true, it means we are updating image on the same cell, so we want to reserve the zoom level & position
   var updatingImageOnSameCell = false
   
+  // enables Live Text analysis & interaction
+  var enableLiveText = false
+  
   var image: UIImage? {
     didSet {
       if image?.imageData != nil, let image = image {
         imageView.setGifImage(image)
       } else {
         imageView.image = image
+        if enableLiveText {
+          if #available(iOS 16, *) {
+            if let image = image {
+              Task {
+                await analyzeImage(image)
+              }
+            }
+          }
+        }
```

Comment:
I think we can simplify this expression and get rid of some of the nesting:
```suggestion
        if #available(iOS 16, *), enableLiveText, let image = image {
          Task {
            await analyzeImage(image)
          }
        }
```